modules/cli/src/main/resources/completions/zsh

```diff
@@ -49,7 +49,7 @@ _coursier() {
   local property=( {--property,-D}'[set Java properties in the generated launcher]:key-value:' )
   local quiet=( {--quiet,-q}'[quiet output]' )
   local repository=( {--repository,-r}'[repository; for multiple repositories, separate with comma and/or add this option multiple times]:repository:' )
-  local reverseTree=( {--reverse-tree,-T}'[print dependencies as an inversed tree; dependees as children]' )
+  local reverseTree=( {--reverse-tree,-T}'[print dependencies as an inverted tree; dependees as children]' )
```

Comment:
I'm not terribly confident about `inverted`. `inversed` isn't a word, `reversed` would be the natural word to use based on the flag `--reverse-tree`.

Reply:
`reversed` would be fine, yes.
```suggestion
  local reverseTree=( {--reverse-tree,-T}'[print dependencies as a reversed tree; dependees as children]' )
```